plugin/mutate/source/dextool/plugin/mutate/backend/database/schema.d

```diff
@@ -411,6 +412,31 @@ struct MutantTimeoutCtxTbl {
     State state;
 }
 
+import dextool.plugin.mutate.backend.database.type : Rationale;
+
+@TableName(markedMutantTable)
+@TablePrimaryKey("st_id")
+struct MarkedMutantTbl {
+    @ColumnName("st_id")
+    long mutationStatusId;
+
+    long id;
```

Comment:
I recommend changing this to mutationId and `@ColumnName("mut_id")` or something. I didn't understand "what" it was an ID so I had to read the source code in standalone.d/markMutant.